thinc/legacy/loss.py

```diff
@@ -65,10 +67,10 @@ def convert_truths(self, truths: TruthsT, guesses: Floats2d) -> Tuple[Floats2d,
                         )
                         raise ValueError(msg)
                     for i, value in enumerate(truths):
-                        if not isinstance(value, str):
+                        if not (isinstance(value, str) or value == None):
```

Comment:
```suggestion
                        if not (isinstance(value, str) or value is None):
```
Python religion.

Reply:
We discussed this a bit more and the check should be against `missing_value`.